general/open-source.md

```diff
@@ -0,0 +1,18 @@
+# 📖 Langley Foxall Open Source
+We've recently began delving into the open source world as a company, by opening up some of the tools developed for
+internal purposes to the open source community.
+
+We love contributions and want our packages to help people, so if you have suggestions for improvements, please file
+an issue on the relevant repository.
+
+##### Laravel
+* [Xero - An Eloquent-esque interface for the Xero PHP Wrapper](https://github.com/langleyfoxall/xero-laravel)
```

Comment:
Perhaps put the full package names here 'Xero Laravel', etc. and link only the package name, not the package description? Might be better for SEO.